Server/Classes/Handlers/VehicleSpawner.sqx

```diff
@@ -125,12 +125,69 @@ namespace Intrusion.Server
 		    clearBackpackCargoGlobal _vehicleObj;
 		    clearItemCargoGlobal _vehicleObj;
 		    
-		    {
-            	_vehicleObj addBackpackCargoGlobal [_x, 1];
-		    } foreach _vehicleClassInfo.CargoWeapons as String;
-		    
         	_intrusionVehicle = [_vehicleObj, _side, _professionType, _group, _cost] new IntrusionVehicle;
         	_vehicleObj setVariable ["IntrusionVehicle", _intrusionVehicle, true];
+        	
+	    	{
+	    		private ["_attachObj" as Object, "_attachObjInfo" as VehicleAttachmentInfo];
+	    		_attachObjInfo = _x;
+	    		
+	    		_attachObj = _attachObjInfo.ClassName createVehicle (getPos _vehicleObj);
+				_attachObj attachTo [_vehicleObj, _attachObjInfo.AttachOffset];
+				clearWeaponCargoGlobal _attachObj;
```

Comment:
From what I can se we are not sure if the attached item is a cargo. What if we attach something else? Skcript error here? Shouldn't it be moved into the if isKindOf Bag_Base below?